MatrixSDK/MXRestClient.h

```diff
@@ -71,20 +71,41 @@ FOUNDATION_EXPORT NSString *const kMXRestClientErrorDomain;
 typedef enum : NSUInteger
 {
     /**
-     "scale" trys to return an image where either the width or the height is smaller than the
+     "scale" tries to return an image where either the width or the height is smaller than the
      requested size. The client should then scale and letterbox the image if it needs to
      fit within a given rectangle.
      */
     MXThumbnailingMethodScale,
 
     /**
-     "crop" trys to return an image where the width and height are close to the requested size
+     "crop" tries to return an image where the width and height are close to the requested size
      and the aspect matches the requested size. The client should scale the image if it needs to
      fit within a given rectangle.
      */
     MXThumbnailingMethodCrop
 } MXThumbnailingMethod;
 
+/**
+ List the filtering options used during the messages search. This option concerns the attachments handling.
```

Comment:
I am not sure about the naming. 'Attachment' has no particular meaning for Matrix events. 'Media' is a bit too specific regarding what contains_url do, ie filtering events which have a 'content' with a 'url' field. Let's discuss it in 15min :)